css/enhancedsteam.css

```diff
@@ -393,6 +393,10 @@ input[type=checkbox].es_dlc_selection:checked + label {
 	color: white;
 }
 
+.emoticon_popup_ctn {
+	z-index: 999;
+}
+
```

Comment:
Gee, I don't really remember, but I think it was because it got behind an element? But if you tested it and it was fine without this, I think we can remove it.

Reply:
I tested it with and without. In both cases I can scroll. But I'll remove it anyway.